cmd/skywire-visor/commands/root.go

```diff
@@ -135,7 +139,15 @@ var rootCmd = &cobra.Command{
 }
 
 // Execute executes root CLI command.
-func Execute() {
+func Execute(ui embed.FS) {
+	uiFS, err := fs.Sub(ui, "static")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	uiAssets = uiFS
```

Comment:
Why are we not just passing in `uiFS` below?

Reply:
@iketheadore `Execute` is a cobra lib function, we can't pass it in there